Updated mathematics docs (#222)
Unity-Technologies/Unity.Mathematics: src/Documentation~/quaternion-multiplication.md

# Quaternion multiplication

> To rotate a quaternion, use the [`AxisAngle`](xref:Unity.Mathematics.quaternion.AxisAngle*) method. You specify the axis of rotation and the rotation around that axis, but supply the axis first and the angle second. Also, all the angles are in radians rather than degrees. `math.mul` multiplies the quaternion, just as with [matrices](4x4-matrices.md) and vectors.

Reviewer: ```suggestion
To rotate a quaternion, use the [`AxisAngle`](xref:Unity.Mathematics.quaternion.AxisAngle*) method. You need to specify the axis of rotation and the angle of rotation, in that order. All are in radians rather than degrees. `math.mul` multiplies the quaternion, just as with [matrices](4x4-matrices.md) and vectors.
```

Reviewer: Do you think we need to tell users how to convert from degrees to radians, or does that fall outside of our remit?

Author: I feel like that's just a general conversion thing like converting inches to centimetres, or F to C etc. So I don't think so!
